Cargo.toml

```diff
@@ -2,6 +2,8 @@
 name = "avl-tree"
 version = "0.1.0"
 authors = ["Gabriela Alexandra Moldovan <gabi_250@live.com>"]
+description = "A very simple AVL tree implementation"
+license = "GPL-3.0-or-later"
```

Comment:
Up to you, obviously, but Rust libraries are generally MIT + Apache-2.0. The GPL will heavily reduce potential useage.

Reply:
This release is just for fun to be perfectly honest [I just wanted to go through the process of publishing a crate :)]. I'll probably extend this project to practice writing Rust.